src/MICore/MICoreResources.resx

```diff
@@ -204,4 +204,7 @@ Error: {1}</value>
   <data name="Status_BreakpointPending" xml:space="preserve">
     <value>Module containing this breakpoint has not yet loaded or the breakpoint address could not be obtained.</value>
   </data>
+  <data name="Error_InvalidLocalCoreDumpPath" xml:space="preserve">
+    <value>Invalid path to core dump file path '{0}'. File must be a valid file name that exists on the Visual Studio computer.</value>
```

Comment:
Visual Studio isn't the only product hosting this. I think I would remove the reference to visual studio here.